.eslintrc.yml

```diff
@@ -2,3 +2,4 @@ extends: 'eslint-config-node-services'
 
 rules: 
   no-bitwise: 0
+  no-use-before-define: 1
```

Comment:
We actually set it up in the `eslint-config-node-services` https://github.com/wikimedia/eslint-config-node-services/blob/master/.eslintrc.js#L60

This means that we allow using classes before they're declared. So if you move the createUuidFromBuffer function to the top of the file, you will be able to delete this line.